runtime/calamari/src/lib.rs

```diff
@@ -173,12 +173,20 @@ parameter_types! {
     pub const SS58Prefix: u8 = manta_primitives::constants::CALAMARI_SS58PREFIX;
 }
 
+parameter_types! {
+    pub UnpausablePallets: Vec<Vec<u8>> = vec![b"Democracy".to_vec(), b"Balances".to_vec(), b"Council".to_vec(), b"CouncilCollective".to_vec(), b"TechnicalCommittee".to_vec(), b"TechnicalCollective".to_vec()];
```

Comment:
Do you think there's a way to no use strings here. Just wondering

Reply:
As extrinsic parameter only accept `Vec<u8>`, the configuration has to be strings. My first thought of this improvement is align to `Call`, while we can config in runtime using `Call`, but inside exstrinsic, there're no other way to convert `Vec<u8>` to `Call`.

Reply:
Non-pausable pallets don't need to be an extrinsic parameter, right? there is no extrinsic like `set_nonpausables` that would need to get the pallet names from the user.

We can just have a Vec<pallet_index_type> in the runtime. We would only update this with runtime upgrades and nothing else

Reply:
`PalletInfo::index::<calamari_runtime::Democracy>` return index of `Democracy` pallet in runtime. however `ensure_can_pause` only accept `pallet_name: Vec<u8>` as its parameter. I don't think `pallet_name` can be convert into `index`, thus they can't comparing. Unless we're changing `palelt_name` parameter type of pause extrinsics to `pallet_index`.

Reply:
Right, i think we can just use the pallet index for pausing - these are not public-use extrinsics anyway.

Using index keeps the code simpler but makes usage harder.
I personally think the tradeoff is worth it ( if we also get rid of individually pausing extrinsics )

Reply:
If we use pallet index and the `PausedTransactions` storage is: `Map(pallet_index, ())`, but the entry point from `Call` is still `pallet_name`, which means there needs to be some way convert `pallet_name` to `pallet_index`. I don't think we have this conversion.
https://github.com/Manta-Network/Manta/blob/0365ac76bacfaf41d5599e2a044dbf572026ae97/pallets/tx-pause/src/lib.rs#L152-L157